xml/System.Text.Json.Serialization/JsonStringEnumConverter.xml

```diff
@@ -84,9 +84,9 @@ Reading is case insensitive. Writing can be customized by using a <xref:System.T
         <Parameter Name="typeToConvert" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeToConvert">The type to be checked.</param>
```

Comment:
I think we should follow the inheritdoc comment and change this to whatever we land on for JsonConverter base type (from https://github.com/dotnet/dotnet-api-docs/pull/2889#discussion_r309970645).

I prefer:

```suggestion
        <param name="typeToConvert">The type of the object to check whether it can be converted by this converter instance.</param>
```

cc @JeremyKuhne, @steveharter